Tools/MiniBrowserSwiftUI/Shared/BrowserTab.swift

```diff
@@ -160,7 +160,9 @@ private struct ExternalURLNavigation : Identifiable, Hashable {
 
 struct BrowserTab_Previews: PreviewProvider {
     static var previews: some View {
-#if os(iOS)
+#if os(macOS)
```

Comment:
Does this not also disable previews on iOS?

Reply:
iOS will be handled by the `#else` portion.